config/routes.rb

```diff
@@ -2,5 +2,6 @@
   devise_for :users
   root to: 'pages#home'
 
+  resources :groups
```

Comment:
don't add all the resources if the controller actions haven't been added yet

Reply:
kk nested and changed to just index rn
![image](https://user-images.githubusercontent.com/3223329/112416309-c53a9c80-8d68-11eb-8368-524efbf961df.png)

Reply:
if you do like this it means the url will be /users/:user_id/groups, meaning we'll get a list of all groups _belonging to a user_
but that's not what we want, what we want is a list of all groups, period (plus the filtering mentioned above)

Reply:
![image](https://user-images.githubusercontent.com/3223329/112792401-5a59d000-909e-11eb-8dbb-0077d6d6c810.png)
I don't know how to add the filtering. Hope this is what you're looking for.